woocommerce/woocommerce-gateway-stripe
# Redirect to settings tab after account is connected

Fixes #2202

## Changes proposed in this Pull Request:

This PR redirects the user to the settings tab to enable Stripe after connecting an account (either by OAuth or entering keys manually).

Previously, users were redirected to the payment methods tab, and they didn't know that Stripe was not activated automatically. We decided not to activate the gateway automatically because we cannot assume the user wants to enable Stripe as soon as they connect an account.

## Testing instructions

On a clean install, go to the Stripe settings page (/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe), and connect an account.

Expected result: the page is redirected to the settings tab after the connection. The `Enable Stripe` checkbox is disabled.

* perform tests with OAuth and Keys entered manually. To test OAuth you need a live site with SSL enabled.

<img width="1064" alt="Screen Shot 2021-12-10 at 13 40 20" src="https://user-images.githubusercontent.com/4153905/145609554-6b787e3a-461e-412f-b5b8-640ade1d3604.png">

---


**Post merge**
